webapp/sources/rudder/rudder-web/src/main/webapp/secure/utilities/eventLogs.html

```diff
@@ -70,21 +74,22 @@ <h1>
     </div>
   </div>
   <script id="rollbackBlock" type="text/template">
-    <div class='rollbackDisplay'>
-      <fieldset class='rollbackFieldSet'>
+    <div class='rollbackDisplay d-flex flex-row-reverse'>
+      <fieldset class='rollbackFieldSet pt-3'>
         <legend>Rollback</legend>
-        <div id='{{rollbackId}}'>
+        <div id='{{rollbackId}}' class="d-flex w-auto align-items-center column-gap-2">
           <span class='alignRollback'>Restore configuration policy to</span>
-          <ul style='display: inline-block;padding-left:5px; text-align:left;'>
-            <input class='radio' type='radio' id='after' name='action' value='after' checked/>
-            <label for='after'>After</label>
-            <input class='radio' type='radio' id='before' name='action' value='before'/>
-            <label for='before'>Before</label>
-          </ul>
+
+          <div class="btn-group" role="group" aria-label="Rollback actions">
+            <input type="radio" class="btn-check" name="action" id="before" value="before" autocomplete="off">
+            <label class="btn btn-outline-secondary" for="before">Before</label>
+
+            <input type="radio" class="btn-check" name="action" id="after" value="after" autocomplete="off" checked>
+            <label class="btn btn-outline-secondary" for="after">After</label>
+          </div>
```

Comment:
changing the class from `radio` to `btn-check` seems to make some JS code fail in rudder-datatables because there is a `$('.radio')` to search for the value : 
![image](https://github.com/user-attachments/assets/7dbba012-cd22-4275-af9f-870ed9160c57)